parity-reconstruct/src/lfdbt.rs

```diff
@@ -0,0 +1,94 @@
+//! Utilities for using [`parity-reconstruct`](crate) with the LoRaWAN Fragmented Data Block standard.
+use bitvec::{array::BitArray, view::BitViewSized};
+
+use crate::ParityMatrix;
+
+/// Paritymatrix specified in the LoRaWAN Fragmented Data Block Transport standard
+pub struct LfdbtParity {
+    n: usize,
+}
+
+impl LfdbtParity {
+    /// Create a LoRaWAN Fragmented Data Block Transport parity matrix
+    pub fn new(n: usize) -> Self {
+        LfdbtParity { n }
+    }
+}
+
+fn prbs23(x: usize) -> usize {
+    let b0 = x & 1;
+    let b1 = (x & 32) >> 5;
+    (x >> 1) + ((b0 ^ b1) << 22)
+}
```

Comment:
Usize is tricky and on embedded devices in Rust it can range from u64 to u16. This doesn't work for u16...
So maybe use a fixed u32 or something?